bats/tests/resources/docker/Dockerfile

```diff
@@ -12,6 +14,6 @@ RUN PATH="$HOME/.cargo/bin:$PATH" RUST_LOG=debug cargo build -q --workspace
 WORKDIR /src
 COPY entrypoint_bootstrap_node.sh entrypoint_no_bootstrap_node.sh entrypoint_auth_node.sh ./
 RUN chmod +x ./*.sh
-RUN git clone https://github.com/pyrsia/pyrsia.git
+RUN git clone --branch $GIT_BRANCH $GIT_REPO
```

Comment:
Root of all evil, the rest of the changes are kind of related (directly or indirectly) to this change.